core/textinput/src/textinput/TerminalDisplayWin.cpp

```diff
@@ -32,24 +32,21 @@ namespace textinput {
     DWORD mode;
     SetIsTTY(::GetConsoleMode(::GetStdHandle(STD_INPUT_HANDLE), &mode) != 0);
 
-    fOut = ::GetStdHandle(STD_OUTPUT_HANDLE);
-    bool isConsole = ::GetConsoleMode(fOut, &fOldMode) != 0;
-    if (!isConsole) {
```

Comment:
This commit seems to suggest that we need to call `CreateFile` even for `isConsole == true`, to fix the redirect. Is that correct? Can you argue why in the commit log? I'm not sure I can read that out of the existing one.

Reply:
If I knew the reason, I would have put it in the log, I found it while trying to fix another issue...

Reply:
FYI, I think I found the real issue.